copter/source/docs/traditional-helicopter-tuning.rst

```diff
@@ -175,19 +103,44 @@ problem. Correct the problem before proceeding with roll and pitch tuning.
 
 * This param name changed in ArduPilot 4.0 and later.
 
+Spool up the heli and hover it
+no more than .25 meters above ground in Stabilize flight mode and test the
+starting tail settings. If the tail seems "loose" and doesn't want to hold
+increase the :ref:`ATC_RAT_YAW_P<ATC_RAT_YAW_P__AC_AttitudeControl_Heli>`. If the tail slowly wags back and forth, the :ref:`ATC_RAT_YAW_P<ATC_RAT_YAW_P__AC_AttitudeControl_Heli>` may be too high. If the tail rapidly shakes side to side reduce the
+:ref:`ATC_ANG_YAW_P<ATC_ANG_YAW_P>`.
+
+In all cases it is not recommended to adjust :ref:`ATC_ANG_YAW_P<ATC_ANG_YAW_P>` below 3.5 or
+:ref:`ATC_RAT_YAW_P<ATC_RAT_YAW_P__AC_AttitudeControl_Heli>` above 0.38. If your helicopter cannot seem to achieve a solid tail
+within those limits you likely have a mechanical problem with the tail - either
+excessive "slop" or play in the linkage, binding of the linkage or a servo
+problem. Correct the problem before proceeding with roll and pitch tuning.
+
+The tune quality can be checked by examining the logs for ATT.DesYaw vs ATT.Yaw as shown below for an imperfect but flyable tune:
+
+.. image:: ../../../images/yaw_tune.jpg
+    :target: ../_images/yaw_tune.jpg
+
+Note that there is a little overshoot and neutral ringing indicating that the FF or P gain is a little high or there is not enough D and could be improved a bit.
+
 Setting _FF and ACCEL_x_MAX Parameters for Desired Pitch and Roll Response
 ==========================================================================
+
 In both pitch and roll axes, the FF gain is set so that the actual aircraft
 rate matches the desired rate. To do this, the RATE message in the log is
-required to compare the P.des and P signals for pitch and the R.des and R
-signals for roll. With the FF gains set to 0.15, takeoff and establish a hover
+required to compare the RATE.PDes and RATE.P log data for pitch and the RATE.RDes and RATE.R
+log data for roll.
+
+With the FF gains set to 0.15, takeoff and establish a hover
 in Stabilize flight mode, then make some sharp stick inputs in both pitch and
 roll. Land and pull the log from the microSD card and look at the signals in
 your ground station software. If the actual rate is more than the desired rate
-then you'll want to decrease FF. If it is less, increase FF. If the desired
-and actual rates are offset by some amount it means that your swash was not
-properly leveled in the setup or the CG is not right.  In this case, just make
-sure the change in rate is similar between desired and actual.  If you get the
+then you'll want to decrease FF. If it is less, increase FF.
+
+**(to be added...FF calculation directly from log data)**
+
```

Comment:
See the [tuning guide](https://drive.google.com/file/d/1BBnF2jPyAN-AXjm39TO_NW3uQFiDw-sZ/view?usp=sharing) that I put together but just haven't put it into this wiki.  If you want to try it and modify the wiki from it, that's fine.  What I wanted you to look at in particular is the VFF tuning on slides 7-9.  You can show the mavexplorer method once they perform this maneuver but the maneuver is what I really wanted you to see.  Tell the user to set the max roll or pitch rate to 50 deg/s and then go fly and perform a maneuver that goes from 30 deg in one direction to 30 deg  in the opposite using full stick in Stabilize mode.  The command model will limit the rate to 50 deg/s and the mavexplorer can be used afterwards to determine the best value of VFF.

Reply:
its exactly the same math as what we do for figuring out pitch ff in VTOL.....but it can be done without holding the full stick for any length of time....you dont have to hold it, just demand it and get it close....the FF is the same at any rate point, its a linear expression..otherwise limiting the max rate at full stick like you suggest would mess with the outcome

....then we take the PID total output (ROut...but we cant use that since ours does not have FF in it, so we graph the sum of the PIDS) * proposed FF and try to match to actual RATE graphs....getting the amplitudes matched by trail of FF values...only take a couple of guesses..that way rate doesnt have to be maxd out and held..I dont think you need to limit the rates, you can just rock it back and forth as hard as you are comfortable with, but its a safe to limit them....but fear limits it pretty well, haha...I dont have any objection to either method, limited rate or no, graphical match or math....my only issue with the math is that it requires eyeballing for a value, whereas overlaying the two graphs could be a bit more precise since the eye is better at declaring a match rather than estimating an extended line to an axis... but am happy to put either method in...perhaps do the math and then re-graph with the FF*ROut*57.3  vs R and graphs should ovelay as a double check on the math and eyeballing of R and ROut values

Reply:
> .the FF is the same at any rate point, its a linear expression..otherwise limiting the max rate at full stick like you suggest would mess with the outcome

Its not a linear response.  We want to match the steady state rate.   Now there is probably some don't let perfection get in the way of good enough factor here.  I prefer they do the maneuver to evaluate the amount of VFF required for steady state response.  and they should probably look at it in both directions.  If they didn't trim the swashplate correctly, then this will show that pretty distinctly.  so either they take the time to trim it properly or use an average of the VFF they found in each direction.